From 0c45c83a787b75d2b9d8550773da5907d8bc0133 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 20 Feb 2017 11:43:26 +0800
Subject: [PATCH 1532/5242] MLK-13997: ARM: dts: disable rpmsg-audio in wm8960
 dts for conflict

commit  9c32534206371d190272276811de9f0e68f3c624 from
https://source.codeaurora.org/external/imx/linux-imx.git

The rpmsg-audio and wm8960-audio use same device in two way, there is
conflict when enabled together.
imx7ulp-evk-wm8960.dts is for demo usage, there is a hardware issue
(TKT320235) that EDMA interrupt can't wake up the A7 core, so there
is glitch noise occassionally.

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm/boot/dts/imx7ulp-evk-wm8960.dts |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts b/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts
index 9400e69..3389d95 100644
--- a/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts
+++ b/arch/arm/boot/dts/imx7ulp-evk-wm8960.dts
@@ -112,6 +112,9 @@
 		};
 	};
 
+	sound-rpmsg {
+		status = "disabled";
+	};
 
 	sound {
 		compatible = "fsl,imx7ulp-evk-wm8960",
-- 
1.7.9.5

